portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,10 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+async function getName() {
+    const response = await fetch('/data');
+      console.log('Handling the response.');
+    const quote = await response.text();
+    document.getElementById("quote").innerHTML = quote;
```

Comment:
I'm going to recommend that you use `.innerText` instead of `.innerHTML` for a few reasons:

- `innerHTML` can open your app to XSS attacks if you're not careful (see https://medium.com/@jenlindner22/the-risk-of-innerhtml-3981253fe217 for example). For this particular exercise, it's not really an issue, but it's something that you should be aware of.
- from an architectural standpoint, it's preferable to fetch raw data from the server, and leave the styling to the UI.

You should be able to pretty easily accomplish the effect you want by adjusting the styling of the "quote" element.

Reply:
Gotcha, thanks!!